tests/php/SubsiteAdminFunctionalTest.php

```diff
@@ -134,16 +149,16 @@ public function testSubsiteAdmin()
             'Is redirected away from forbidden section');
 
         // Check forbidden site, on a section that's allowed on another subsite
-        $this->getAndFollowAll("admin/pages/?SubsiteID=0");
+        $this->getAndFollowAll('admin/pages/?SubsiteID=0');
```

Comment:
Perhaps we should enforce admin as the root URL in the setUp method just for brevity - `Config::modify()->set(AdminRootController::class, 'url_base', 'admin')` in case it has been changed in user code. What do you think?